### DISCUSSION TITLE
Global Circulation Models (GCMs)

### DISCUSSION THREAD

**Discussion by LindeNozomiLeo** [biomodhub/biomod2#277] [Q&A] [answered]
Hi,

I'm new to biomod2 and have been reading a lot of papers that use some sort of ensemble modeling and I'm having a hard time finding out how people use the different GCMs of climate scenarios (as provided by CHELSA etc.)  to get a single output for a future climate scenario. 
Is there any recommendation for how to get a single output per scenario, but still use the 5 different GCMs available?  or can they be ensembled in some way by tweaking a function?
I believe some people say that if you would average them before doing the SDM that they would lose the information that they hold.
Is anyone working with GCMs from CHELSA that can give me some insights? help would be greatly appreciated.
Thank you in advance!

**Comment by rpatin** [comment, marked as answer]
Hi,
I can try to add some insight on the topic.
I agree that it would feel wrong to average the GCM before doing the SDM. 
In our current modeling project, we plan on building 5 different sets of SDM, one for each GCM. I don't think we plan on having a single output at the end but will just compare the output. However if you would like to have a single output, I guess you could ensemble your projection (using mean, coefficient of variation, ...) to highlight the consistency and discrepancy of your modeling output given the GCM.
_Note that you would need to build the ensemble model manually, as `biomod2` only manage ensemble with common environmental data._ 
Hope this helps,
Best regards,
Rémi

**Reply by LindeNozomiLeo** [reply]
Thank you! that answers a lot of the questions I had!
